BoxContentSDK/BoxContentSDK/Models/BOXFile.m

```diff
@@ -79,6 +80,20 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
         if (lock && ![lock isKindOfClass:[NSNull class]]) {
             self.lock = [[BOXFileLock alloc] initWithJSON:lock];
         }
+        
+        // Parse Representations.
+        NSDictionary *representationsJSON = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyRepresentations
+                                                                          inDictionary:JSONResponse
+                                                                       hasExpectedType:[NSDictionary class]
+                                                                           nullAllowed:NO];
```

Comment:
I think it should be:
nullAllowed:YES suppressNullAsNil:YES

Could probably do the same for almost all properties in BOXRepresentation